docs/installation.rst

```diff
@@ -43,17 +43,17 @@ Python 3.8+ and pysat 3.0.4+.
     A. Install on the system (root privileges required)::
 
 
-         sudo python setup.py install
+         sudo pip install .
 
     B. Install at the user level::
 
 
-         python setup.py install --user
+         pip install --user .
 
     C. Install with the intent to develop locally::
 
 
-         python setup.py develop --user
+         pip install -e --user .
```

Comment:
Though this is broken in many instances.

Reply:
Wait, pip install -e is broken in many cases?

Reply:
Yes, it doesn't work with a meson back end (see apexpy).  Not sure about other things.

Reply:
vexing

Reply:
Yup, one of the reasons I am still annoyed about the setuptool deprecations.